### PR TITLE
Internal change.

### PR DESCRIPTION
**Pull request by joshh-devrel**
Internal change.

### CODE REVIEW

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java [hunk 1]
```diff
@@ -53,10 +50,12 @@ public class VungleMediationAdapter extends RtbAdapter
...
   public static final String KEY_APP_ID = "appid";
 
+  private VungleRtbInterstitialAd rtbInterstitialAd;
+  private VungleRtbRewardedAd rtbRewardedAd;
+
   private AdConfig mAdConfig;
   private String mUserID;
   private String mPlacement;
-  private String mAdMarkup;
   private final Handler mHandler = new Handler(Looper.getMainLooper());
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java:59]: mHandler can be removed.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java [hunk 2]
```diff
@@ -66,17 +63,6 @@ public class VungleBannerAdapter implements PlayAdCallback {
...
-   * Vungle listener class to forward to the bidding adapter.
-   */
-  private MediationAdLoadCallback<MediationBannerAd, MediationBannerAdCallback> mediationAdLoadCallback;
-  private MediationBannerAdCallback mediationBannerAdCallback;
-
-  /**
-   * Bid response of Bidding unit.
-   */
-  private String mAdMarkup;
-
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:79]: Comment here but not relate to this line, we can remove this useless constructor:
```
public VungleBannerAdapter(@NonNull String placementId, @NonNull String uniqueRequestId,
      @NonNull AdConfig adConfig, @NonNull MediationBannerAd mediationBannerAd)
```

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java [hunk 3]
```diff
@@ -255,17 +227,12 @@ public void onError(String id, VungleException exception) {
             mediationListener.onAdFailedToLoad(mediationAdapter, error);
             return;
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:228]: We can remove this `return;` code line.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java [hunk 4]
```diff
@@ -298,10 +265,6 @@ private void createBanner() {
           mediationListener.onAdLoaded(mediationAdapter);
           return;
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:266]: We can remove this `return;` code line.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java [hunk 5]
```diff
@@ -312,10 +275,6 @@ private void createBanner() {
           mediationListener.onAdFailedToLoad(mediationAdapter, error);
           return;
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:276]: We can remove this `return;` code line.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java [hunk 6]
```diff
@@ -327,10 +286,6 @@ private void createBanner() {
         mediationListener.onAdFailedToLoad(mediationAdapter, error);
         return;
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:287]: We can remove this `return;` code line.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java [hunk 7]
```diff
@@ -392,10 +345,6 @@ public void onAdClick(String placementID) {
       mediationListener.onAdOpened(mediationAdapter);
       return;
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:346]: We can remove this `return;` code line.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java [hunk 8]
```diff
@@ -409,9 +358,6 @@ public void onAdLeftApplication(String placementID) {
       mediationListener.onAdLeftApplication(mediationAdapter);
       return;
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:359]: We can remove this `return;` code line.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java [hunk 9]
```diff
@@ -422,15 +368,10 @@ public void onError(String placementID, VungleException exception) {
       mediationListener.onAdFailedToLoad(mediationAdapter, error);
       return;
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:369]: We can remove this `return;` code line.

**Review by HuFangshuai — APPROVED** [review]
Overall looks good to me, with only some minor comments.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java (hunk 1 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java]: Done.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/rtb/VungleRtbRewardedAd.java [hunk 10]
```diff
@@ -0,0 +1,266 @@
...
+  private final MediationAdLoadCallback<MediationRewardedAd, MediationRewardedAdCallback> mMediationAdLoadCallback;
+  @Nullable
+  private MediationRewardedAdCallback mMediationRewardedAdCallback;
+
+  private AdConfig mAdConfig;
+  private String mPlacement;
+  private String mAdMarkup;
+  private String mUserID;
+
+  private final Handler mHandler = new Handler(Looper.getMainLooper());
```
[the remark this answers is not in the document]
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/rtb/VungleRtbRewardedAd.java]: Done.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java (hunk 2 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:79]: Done.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java (hunk 3 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:228]: Done.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java (hunk 4 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:266]: Done.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java (hunk 5 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:276]: Done.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java (hunk 6 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:287]: Done.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java (hunk 7 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:346]: Done.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java (hunk 8 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:359]: Done.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java (hunk 9 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:369]: Done.

**Pull request closed without merging**